heal.js

```diff
@@ -25,11 +25,21 @@ var ClusterManager = require('./lib/cluster.js');
 var assertNoError = require('./lib/util.js').assertNoError;
 var program = require('commander');
 
+function safeParseInt(number, defaultValue) {
+    number = parseInt(number, 10);
+    if (isNaN(number)) {
+        return defaultValue;
+    } else {
+        return number;
+    }
+}
+
 function main() {
     program
         .description('Start a partition heal coordinated by the target node')
         .option('--tchannel-v1')
-        .option('--verbose')
+        .option('--quiet')
```

Comment:
I like commands that don't print anything when it succeeds so I implemented it with `--verbose` before, but I can imagine that most people would like to see output in normal operation.